src/components/Charts/Redesign/NewChartBlock.tsx

```diff
@@ -14,23 +16,38 @@ import {
 } from 'components/Charts/Redesign/Groupings';
 import { MetricValues } from 'common/models/Projections';
 import ChartFooter from 'components/NewLocationPage/ChartFooter/ChartFooter';
+import { Metric } from 'common/metricEnum';
 
 const NewChartBlock: React.FC<{
   isMobile: boolean;
   region: Region;
   stats: MetricValues;
   projections: Projections;
   group: ChartGroup;
-}> = ({ projections, stats, group, region }) => {
+  clickedStatMetric: Metric | null;
+  groupRef: React.RefObject<HTMLDivElement>;
+}> = ({ projections, stats, group, region, groupRef, clickedStatMetric }) => {
   const { metricList, groupHeader } = group;
 
   const [activeTabIndex, setActiveTabIndex] = useState(0);
   const onChangeTab = (newTabIndex: number) => {
     setActiveTabIndex(newTabIndex);
   };
 
-  const inactiveTabs = metricList.length === 1;
-  const TabsWrapper = inactiveTabs ? InactiveTabWrapper : ChartTabs;
+  // Selects correct tab after clicking summary stat + scrolling to chart block
+  useEffect(() => {
+    if (!isNull(clickedStatMetric)) {
+      const clickedStatMetricIndex = findIndex(
+        metricList,
+        item => item.metric === clickedStatMetric,
+      );
+      if (clickedStatMetricIndex >= 0) {
+        setActiveTabIndex(clickedStatMetricIndex);
+      }
+    }
+  }, [metricList, clickedStatMetric]);
```

Comment:
So I think there's an edge case that doesn't quite work...  If I:
1) Click "Infection rate" at the top of the page to focus the infection rate tab
2) Click "Daily new cases" inside the chart to focus the daily new cases tab
3) Click "Infection rate" at the top *again* to focus the infection rate tab again

It scrolls to the chart but the tab doesn't change to "infection rate" since `clickedStatMetric` didn't change so this `useEffect` doesn't get run...

To be honest, this is kinda' convoluted so I'm fine with shipping like this if we don't know of an easy fix.  The solutions that come to mind off the top of my head are:
1. Move the `activeTabIndex` state up a level. I sorta' suspect this is the "correct" fix but it seems pretty cumbersome (and maybe there's some better pattern I don't know about).
2. Hacky but I wonder if we could pass in a counter or timestamp or something in addition to `clickedStatMetric` that would change each time the metric is clicked, in order to trigger this effect to run again.

Reply:
i originally had `activeTabIndex` at the `ChartsHolder` level but then realized that when all of the `chartBlocks` share a single `activeTabIndex`, switching a tab in one block switches the tab in all of the blocks // they all change in unison. so to keep each block functioning independently, i moved `activeTabIndex` down into the individual blocks

Reply:
an individual activeTabIndex/setActiveTabIndex at the chartsHolder level passed to each block (like the map of block-specific refs) could work

Reply:
but there might be a better way. adding a todo and gonna think on this

Reply:
Sounds good. Yeah, I was imagining something like the block-specific refs, but TODO sounds good for now!